librz/flag/flag.c

```diff
@@ -841,6 +841,19 @@ RZ_API void rz_flag_unset_all(RzFlag *f) {
 	new_spaces(f);
 }
 
+/* unset all flag items in the space with the given name */
```

Comment:
Please add a doxygen documentation for all new RZ_API functions.

Reply:
Okay f77155b8be4fe0533b15ed5a4e51dcc9eefd5e72